.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    target-branch: "dependabot"
+    schedule:
+      interval: "weekly"
```

Comment:
Shouldn't we instantly update packages if any vulnerable versions detected? (I haven't written Dependabot configs and don't know how this works.)

Reply:
Set interval to daily?

Reply:
> Set interval to daily?

Let's leave `weekly`